packages/components/src/dividers/new-messages/new-messages.tsx

```diff
@@ -0,0 +1,28 @@
+import { Stack } from '@tamagui/core'
+import { LinearGradient } from '@tamagui/linear-gradient'
+
+import { Text } from '../../text'
+
+import type { ColorTokens } from '@tamagui/core'
+
+type Props = {
+  color: ColorTokens
+}
+
+const NewMessages = (props: Props) => {
```

Comment:
I'm thinking if it's ideal naming, if it's just a divider and it doesn't actually contain messages, should that be named like `DividerNewMessages` same as `DividerLine` and `DividerDate`?

Reply:
I like that.

Reply:
Done in https://github.com/status-im/status-web/pull/360/commits/035485febcaa26c80893dd2a5e68c32f13100d96

Reply:
(I would even prefer "unread" instead of "new," but let's keep it close to the name in Figma)